app/main/views/users.py

```diff
@@ -65,32 +66,61 @@ def download_users(framework_slug):
         "application_result",
         "framework_agreement"
     ]
-    # insert header column
-    supplier_rows.insert(0, {header: header for header in supplier_headers})
 
-    def iter_csv(rows):
+    formated_rows = []
+    for row in supplier_rows:
+        formated_rows.append([row[heading] for heading in supplier_headers])
+    formated_rows.insert(0, supplier_headers)
 
-        class Line(object):
-            def __init__(self):
-                self._line = None
+    return Response(
+        csv_generator.iter_csv(formated_rows),
+        mimetype='text/csv',
+        headers={
+            "Content-Disposition": "attachment;filename=users-{}.csv".format(framework_slug),
+            "Content-Type": "text/csv; header=present"
+        }
+    )
 
-            def write(self, line):
-                self._line = line
 
-            def read(self):
-                return self._line
+@main.route('/users/download/buyers', methods=['GET'])
+@login_required
+@role_required('admin')
+def download_buyers_and_briefs():
+    buyers = (user for user in data_api_client.find_users_iter(role='buyer'))
+    briefs = (brief for brief in data_api_client.find_briefs_iter(with_users=True))
+    buyer_headings = [
+        "name",
+        "emailAddress",
+        "phoneNumber",
+        "createdAt",
+        "briefs"
+    ]
+
+    buyers_dict = {}
+    for buyer in buyers:
+        buyer['briefs'] = []
+        buyers_dict[buyer['id']] = buyer
+
+    for brief in briefs:
+        for user in brief['users']:
+            brief_string = '{} - {}'.format(brief['title'], brief['status'])
```

Comment:
Is this all that we want to know about the briefs themselves?  There's no `id` needed?